site/en/docs/web-platform/conditional-focus/index.md

```diff
@@ -0,0 +1,185 @@
+
+```js
+// Retain focus if capturing a tab dialed to example.com.
+// Focus anything else.
+const origin = track.getCaptureHandle().origin;
+if (displaySurface == "browser" && origin == "https://example.com") {
+  controller.setFocusBehavior("no-focus-change");
+} else if (displaySurface != "monitor") {
+  controller.setFocusBehavior("focus-captured-surface");
+}
+```
+
+It is even possible to decide whether to focus before calling `getDisplayMedia()`.
+
+```js
+// Focus the captured tab or window when capture starts.
+const controller = new CaptureController();
+controller.setFocusBehavior("focus-captured-surface");
+
+// Prompt the user to share their screen.
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
+```
+
+You can call `setFocusBehavior()` arbitrarily many times before the promise resolves, or at most once immediately after the promise resolves. The last invocation overrides all previous invocations.
+
+More precisely:
+- The `getDisplayMedia()` returned promise resolves on a microtask. Calling `setFocusBehavior()` after that microtask completes throws an error.
+- Calling `setFocusBehavior()` more than a second after capture starts is no-op.
+
+That is, both of the following snippets will fail:
+
+```js
+// Prompt the user to share their screen.
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
+
+// Too late, because it follows the completion of the task
+// on which the getDisplayMedia() promise resolved.
+// This will throw.
+setTimeout(() => {
+  controller.setFocusBehavior("focus-captured-surface");
+});
+```
+
+```js
+// Prompt the user to share their screen.
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
+
+const start = new Date();
+while (new Date() - start <= 1000) {
+  // Idle for ≈1s.
+}
+
+// Because too much time has elapsed, the browser will have
+// already decided whether to focus.
+// This fails silently.
+controller.setFocusBehavior("focus-captured-surface");
+```
+
+Calling `setFocusBehavior()` also throws in the following cases:
+
+- the video track of the stream returned by `getDisplayMedia()` is not ["live"].
+- after the `getDisplayMedia()` returned promise resolves, if the user shared a screen (not a tab or a window).
+
+## Sample  {: #sample }
+
+You can play with Conditional Focus by running the [demo] on Glitch. Be sure to [check out the source code].
+
+{% Glitch { id: 'conditional-focus', path: 'index.js', previewSize: 0 } %}
+
+## Feature detection {: #feature-detection }
+
+To check if `CaptureController.setFocusBehavior()` is supported, use:
+
+```js
+if (
+  "CaptureController" in window &&
+  "setFocusBehavior" in CaptureController.prototype
+) {
+  // CaptureController.setFocusBehavior() is supported.
+}
+```
+
+## Feedback {: #feedback }
+
+The Chrome team and the web standards community want to hear about your experiences with Conditional Focus.
```

Comment:
Should we leave our emails? (Or maybe just mind if you don't want to do that?) Or is this not generally done?

Reply:
We don't do that usually. See examples:
- https://developer.chrome.com/docs/web-platform/region-capture/#feedback
- https://developer.chrome.com/docs/web-platform/capture-handle/#feedback
- https://developer.chrome.com/docs/web-platform/screen-sharing-controls/#feedback